src/helper/HelperApp.cpp

```diff
@@ -50,6 +51,9 @@ namespace SDDM {
         qInstallMessageHandler(HelperMessageHandler);
 
         QTimer::singleShot(0, this, SLOT(setUp()));
+#if defined(Q_OS_LINUX)
+        prctl(PR_SET_CHILD_SUBREAPER);
```

Comment:
According to the manpage:

```
       PR_SET_CHILD_SUBREAPER (since Linux 3.4)
              If arg2 is nonzero, set the "child subreaper" attribute of the calling process; if arg2 is zero, unset the attribute.
```

So this is an invalid call as the argument is missing.